src/rgw/rgw_rados.cc

```diff
@@ -1496,7 +1496,7 @@ int RGWZoneParams::create(bool exclusive)
   }
 
   r = RGWSystemMetaObj::create(exclusive);
-  if (r < 0) {
+  if (r < 0 && r != -EEXIST) {
```

Comment:
this will cause error when create_default races with other create_default, it is up to the caller to handle EEXIST errors